code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -454,6 +454,14 @@
 	var/our_pressure = air_contents.return_pressure()
 	var/our_temperature = air_contents.return_temperature()
 
+	if(our_pressure > pressure_limit + TANK_FRAGMENT_PRESSURE)
+		var/range = (our_pressure - (pressure_limit + TANK_FRAGMENT_PRESSURE)) / TANK_FRAGMENT_SCALE
```

Comment:
This scaling means that very high temperatures will cause much more damage then lower ones, no matter the pressure limit.